src/intl/en/learn-quizzes.json

```diff
@@ -613,5 +613,50 @@
   "defi-5-c-label": "Bitcoin",
   "defi-5-c-explanation": "This is incorrect. Bitcoin is a simple network for storing value, not for running advanced programs. DeFi requires a more flexible system, like Ethereum, that can run complex programs to handle loans and trades automatically.",
   "defi-5-d-label": "Traditional financial institutions",
-  "defi-5-d-explanation": "This is incorrect. DeFi apps don’t need traditional financial institutions. They use blockchain programs called smart contracts to handle transactions automatically."
+  "defi-5-d-explanation": "This is incorrect. DeFi apps don’t need traditional financial institutions. They use blockchain programs called smart contracts to handle transactions automatically.",
+  "smart-contracts-1-prompt": "How are smart contracts characterized?",
+  "smart-contracts-1-a-label": "Smart contracts are just like real (paper) contracts, but stored digitally on the blockchain to save the content securely.",
```

Comment:
Would still suggest "legal contracts" here...

> "like real (paper) contracts"

Just noting that "real" feels ambiguous compared to "legal", and not all contracts need to be on paper either, ie verbal contracts